CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 
 ### Changed
 - Replace switchable controller actions with separate plugins (#575)
+- Controller actions return ResponseInterface (#597)
```

Comment:
Please move the entry to the top of the section so the changes are in reverse chronological order ("newest on top").

Please use the same format in the changelog as in the commit message subject, i.e., use the imperative:

Return `ResponseInterface` in controller actions

For details on improving git commit message, I can highly recommend this article: https://cbea.ms/git-commit/